ds_judgements_public_ui/sass/includes/_govuk_overrides.scss

```diff
@@ -168,3 +168,109 @@
 .govuk-form-group {
   margin-bottom: 0;
 }
+
+.govuk-header__container {
+  position: relative;
+  margin-bottom: -10px;
+  padding-top: 10px;
+  border-bottom: none;
+  display: flex;
+  justify-content: space-between;
+}
+
+.govuk-header {
+  border-bottom: 10px solid transparent;
+  font-family: $font-roboto;
+  font-size: 1.1875rem;
+}
+
+.govuk-header__logo {
+  display: flex;
+  width: 100%;
+  @media only screen and (max-width: $grid-breakpoint-medium) {
+    flex-direction: column;
+    padding-right: 14rem;
+    padding-top: 1.25rem;
+  }
+}
+
+.govuk-frontend-supported .govuk-header__menu-button {
+  display: block;
+  right: -2rem;
+  font-size: 1.2rem;
+  padding-top: 1.25rem;
+  &:focus {
+    outline: none;
+    background-color: $color-black;
+    box-shadow: none;
+    color: $color-white;
+  }
+}
+
+.govuk-header__navigation-list {
+  margin-left: 15rem;
```

Comment:
I noticed we have a large margin left here - we could probably position these header items with flex box to avoid this. This would allow mobile styling to be pretty easy